src/JuMP.jl

```diff
@@ -909,9 +909,9 @@ include("print.jl")
 # Deprecations
 include("deprecated.jl")
 
-getvalue{T<:JuMPTypes}(arr::Array{T}) = map(getvalue, arr)
+getvalue{T<:JuMPTypes}(arr::AbstractArray{T}) = map(getvalue, arr)
```

Comment:
It's been previously discussed. The conclusion for now is to allow ``getvalue`` on objects that JuMP creates, and JuMP can create ``Array{Variable}``, e.g., PSD matrices.